doc/user/layouts/partials/create-source/envelope/debezium/details.html

```diff
@@ -8,6 +8,9 @@
 - [MySQL](https://debezium.io/documentation/reference/0.10/connectors/mysql.html)
 - [PostgreSQL](https://debezium.io/documentation/reference/0.10/connectors/postgresql.html)
 
+Note that Materialize currently does not support truncation for upstream sources. ([7277](https://github.com/MaterializeInc/materialize/discussions/7276)
```

Comment:
I don't think we need to mention it in other places other than here. In the end, this is an edge-case.